spec/models/user_spec.rb

```diff
@@ -9,13 +9,33 @@
   end
   subject { build_stubbed :user }
 
-  it { should have_one :subscription }
+  it "includes Filterable module" do
+    expect(User.ancestors).to include(Filterable)
+  end
+
+  context "associations" do
+    it { should have_one(:subscription).autosave(true) }
```

Comment:
I would use the `expect` syntax instead of `should`. You could rewrite the following as

```ruby
it { is_expected.to have_one(:subscription).autosave(true) }
```